config/credentials.yml.enc.example

```diff
@@ -0,0 +1,8 @@
+aws:
```

Comment:
I would prefer to have environment specific credentials, so for example:
```
development:
  aws:
    service: S3
    access_key_id: "valid access key id"
    secret_access_key: "valid secret access key"
    ....and so on

  allowed_origin: http://localhost:3000

production:
  aws:
    service: S3
    access_key_id: "valid access key id"
    secret_access_key: "valid secret access key"
    ....and so on

  allowed_origin: http://lingoqauploads.example.com
```

Reply:
@rahulbajaj0509  Credentials were the same way before as in the above example. But it was mentioned that "allowed- origins" should not come under any environment specific variable so I removed it. So shall I specify environment specific variables for only "AWS credentials" or even for "allowed-origins". Please do let me know.

Reply:
You are correct. How about we use the rails suggested way for storing credentials environment specific.
Quoting from Rails documentation:
```
The `credentials` command supports passing an `--environment` option to create an
environment specific override. That override will take precedence over the
global `config/credentials.yml.enc` file when running in that environment. So:
 
   rails credentials:edit --environment development
 
will create `config/credentials/development.yml.enc` with the corresponding
encryption key in `config/credentials/development.key` if the credentials file
doesn't exist.
```
Here is a small blog explaining how to do it:
https://blog.bigbinary.com/2019/07/03/rails-6-adds-support-for-multi-environment-credentials.html
 
Note: Since the default environment is development, all we need to do right now is simply copy the contents from credentials.yml.enc file into `config/credentials/development.yml.enc` and delete the credentials.yml.enc file. When we deploy and configure other environments, we will take care to introduce other files also.

Reply:
Perfect!  thanks for looking into this @aaditi-jain :)